main.py

```diff
@@ -32,11 +32,8 @@
         ONLY_INFERENCE=False,
         EPOCHS=50,
         ROLE='Car1',
-        EXPLORATION_EXPLOTATION_THRESHOLD=2500,
-        TIME_BETWEEN_STEPS=0.05,
-        LEARNING_RATE=0.0000001,
-        N_STEPS=200,
-        BATCH_SIZE=200,
+        EXPLORATION_EXPLOTATION_THRESHOLD=50,
+        MODEL_TYPE='A2C',
```

Comment:
Make 'A2C' as param in config, and use it.

Reply:
Done